packages/server/lib/hooks/hooks.ts

```diff
@@ -78,8 +79,7 @@ export const connectionCreated = async (
     const { connection, environment, auth_mode } = createdConnectionPayload;
 
     if (options.initiateSync === true && !isHosted) {
-        const syncClient = await SyncClient.getInstance();
-        await syncClient?.initiate(connection.id as number, logContextGetter);
+        await orchestratorService.createSyncForConnection(connection.id as number, logContextGetter, orchestrator);
```

Comment:
move the `initiate` function out of `syncClient` since it will be necessary to move/refactor it when getting rid of temporal